### ISSUE TITLE
Question of Different Training Objectives

### ISSUE THREAD

**Issue by lavinal712** [MonoFormer/MonoFormer#2]
Thanks for your work!

Your training objectives includes lm loss and diff loss. How is the training results? Is it difficult to train?

**Comment by ZhaoChuyang** [comment]
Hi,
The training consists of both lm loss and diffusion loss, it is important to balance these two losses. Since diffusion is trained from scratch and the LLM is initialized from a pretrained model, so we set the loss weight of lm loss to be relatively small to make the training process stable. You can check the configuration file for training details.
